### DISCUSSION TITLE
Repeated scheduled videos Hero 12

### DISCUSSION THREAD

**Discussion by batscience** [gopro/labs#784] [General]
Hi!

I would like to set my Hero 12 to record 1-minute continuous video at 15 minute intervals between 07:00 - 19:00, and repeat this command every day. I need it to record continuous video rather than time-lapse images put together into a video, and for the camera to switch off in between recordings to save battery. 

I have the GoPro labs firmware installed, but I am having trouble figuring out how to program it to do what I need it to do. 

I was wondering if anyone has had any success doing a similar thing, and/or whether anyone might have any advice on how to achieve this? 

Any help/advice would be greatly appreciated!

Thank you in advance!

**Comment by dnewman-gpsw** [comment]
Converted to a discussion, as you have not found an "Issue" with the Labs, just learning.  Issues are for bug to fix.

While there is a lot of discuss with similar questions, do something like this:
```dV<07:00>19:00!07:00N!S!58E!840N!R```

**dV** - default Video (so the camera boot in video mode)
**<07:00>19:00!07:00N**  - if time < 7am or >7pm, sleep until 7am
**!S** - Start
**!58E** - End after 58s, this will capture a minute.
**!840N** - Sleep 14 minutes
**!R** - repeat it all.

**Reply by batscience** [reply]
Ah, sorry about that - it is my first time on GitHub. Thank you so much for your help!
